docs/source-pytorch/common/lightning_module.rst

```diff
@@ -440,15 +446,17 @@ There are two ways to call ``predict()``:
 .. code-block:: python
 
     # call after training
-    trainer = Trainer()
-    trainer.fit(model)
+    trainer = pl.Trainer()
+    trainer.fit(model=model, train_dataloaders=dataloader)
 
     # automatically auto-loads the best weights from the previous run
     predictions = trainer.predict(dataloaders=predict_dataloader)
 
     # or call with pretrained model
-    model = MyLightningModule.load_from_checkpoint(PATH)
-    trainer = Trainer()
+    model = LightningTransformer.load_from_checkpoint(PATH)
+    dataset = pl.demos.WikiText2()
```

Comment:
@Borda This doesn't work.
@JustinGoheen Could you please revert it in #18452?

Reply:
this is changed in [6671aa6](https://github.com/Lightning-AI/lightning/pull/18452/commits/6671aa6215041d1f8598854e39fd7a5c5330cb69)